osu.Game/OsuGameBase.cs

```diff
@@ -174,8 +174,10 @@ private void load()
             // this adds a global reduction of track volume for the time being.
             Audio.Track.AddAdjustment(AdjustableProperty.Volume, new BindableDouble(0.8));
 
-            dependencies.CacheAs<BindableBeatmap>(beatmap);
-            dependencies.CacheAs<IBindableBeatmap>(beatmap);
```

Comment:
Again, if you're going to do this, make sure to delete the `BindableBeatmap` and `IBindableBeatmap` classes/interfaces.

Reply:
`BindableBeatmap` still needs to exist for test cases and `IBindableBeatmap` is already gone.